src/platforms/javascript/guides/svelte/features/componenttracking.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Track Svelte Components
+---
+
+Sentry's Svelte SDK offers a feature to monitor the performance of your Svelte components: **Component Tracking**. By enabling this feature, you will get spans in your transactions showing the initialization and update cycles of your Svelte components. This feature allows you to get a drilled down view into how your components are behaving, to e.g. identify slow initializations or frequent updates which might have an impact on your app's performance.
+
+## Usage
+
+To get started, simply add the Sentry `componentTrackingPreprocessor` to your `svelte.config.js` file:
+
+```javascript {tabTitle: svelte.config.js}
+import * as Sentry from "@sentry/svelte";
+
+const config = {
+  preprocess: [
+    Sentry.componentTrackingPreprocessor({
+      // Add the components you want to be tracked to this array.
+      // Specificy `true` to track all components or `false` to disable
+      // tracking entirely
+      // (defaults to `true`)
+      trackComponents: ["Navbar", "PrimaryButton", "LoginForm"],
+      // To disable component initialization spans, set this to `false`.
+      // (defaults to `true`)
+      trackInit: true,
+      // To disable component update spans, set this to `false`
+      // (defaults to `true`)
+      trackUpdates: false,
+    }),
+    // ...
+  ],
+  // ...
+};
+
+export default config;
+```
+
+This preprocessor is called at application build time. It inserts a function call to a function in the Sentry SDK into the `<script>` tag of your components before it is compiled and bundled. The called function takes care of recording the spans by using the Svelte component's lifecycle hooks.
```

Comment:
Before what is compiled and bundled? The app? There's already an "it" in this sentence for the preprocessor, so we want to be specific.

```suggestion
This preprocessor is called at application build time. It inserts a function call to a function in the Sentry SDK into the `<script>` tag of your components before your app is compiled and bundled. The called function takes care of recording the spans by using the Svelte component's lifecycle hooks.
```